src/buffer.rs

```diff
@@ -26,6 +26,7 @@ pub enum Type {
     MetaOutput          = 14,
 
     /// Deprecated, do not use
+    #[deprecated]
     Private             = 0x80,
```

Comment:
I feel like this may not be useful. If you have a better reason than "do not use", it'd work much better here! :D


```suggestion
    #[deprecated(note = "do not use")]
    Private             = 0x80,
```